Project.toml

```diff
@@ -15,21 +15,18 @@ DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 Insolation = "e98cc03f-d57e-4e3c-b70c-8d51efe9e0d8"
 JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 NCDatasets = "85f8d34a-cbdd-5861-8df4-14fed0d494ab"
-OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
-SimpleNonlinearSolve = "727e6d20-b764-4bd8-a329-72de5adea6c7"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 SurfaceFluxes = "49b00bb7-8bd4-4f2b-b78c-51cd0450215f"
 TempestRemap_jll = "8573a8c5-1df0-515e-a024-abad257ee284"
-TerminalLoggers = "5d786b92-1e48-4d6f-9151-6b4477ca9bed"
 Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
 [compat]
 CLIMAParameters = "0.4, 0.5, 0.6, 0.7"
-ClimaAtmos = "=0.16.2"
+ClimaAtmos = "0.16"
 ClimaComms = "0.5.6"
```

Comment:
Is this patch spec necessary?

Reply:
Yes, to help with some of the MPI/HDF5 problems that came up last week

Reply:
It should be `0.16.3`. The code won't work with any previous version of `ClimaAtmos`.

Reply:
I meant the the ClimaComms dep :)

Reply:
Ahh good point, I do need to specify the ClimaAtmos patch (oops)